src/names.rs

```diff
@@ -19,6 +19,7 @@ pub static KNOWN_NAMES: phf::Map<u64, &'static str> = phf::phf_map! {
     359950109229973504u64 => "ManganeseFrog",
     368635452925935616u64 => "TopHatimus",
     373610438560317441u64 => "Matt",
+    113455311058108416u64 => "bowlercaptain"
```

Comment:
put your ID in order yo

Reply:
Is... is this list expected to be sorted by ID number? Is that a benefit? Because semantically I would expect to place the newest entry at the end of the list, regardless of hash key.

Reply:
Yeah rustfmt will complain, which you were supposed to magically know through telepathy or something to run it...